tasks/lint.rake

```diff
@@ -94,4 +94,16 @@ namespace :lint do
 
     FixmeLinter.new.run
   end
+
+  desc "RSpec specs for linting project files"
+  task :rspec do
+    puts "Linting project files..."
+
+    abort unless system(
+      "rspec",
+      "spec/gemfiles_spec.lint.rb",
+      "spec/changelog_spec.lint.rb",
+      "spec/i18n_spec.lint.rb"
+    )
```

Comment:
These specs are actually linting files, so I moved them to the lint task as well. The rename is so that they don't get run by default with the regular `rspec` run.